Solutions/AzureDevOpsAuditing/Data Connectors/_createUiDefinition.json

```diff
@@ -0,0 +1,86 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/0.1.2-preview/CreateUIDefinition.MultiVm.json#",
+    "handler": "Microsoft.Azure.CreateUIDef",
+    "version": "0.1.2-preview",
+    "parameters": {
+        "config": {
+            "isWizard": true,
+            "basics": {
+                "description": "<img src=\"https://raw.githubusercontent.com/Azure/Azure-Sentinel/master/Logos/Azure_Sentinel.svg\" width=\"75px\" height=\"75px\">\n\n**Azure DevOps Auditing - Connector** \n\nThe Azure DevOps audit log connector provides the capability to ingest Azure DevOps auditing logs into Microsoft Sentinel. \nBy connecting Azure DevOps audit logs into Microsoft Sentinel, you can view this data in workbooks, use it to create custom alerts, and improve your investigation process. \n\n**Information:** \nIn order to configure Azure DevOps the deployment uses an ARM deployment script which requires a user assigned identity. \nYou can remove this managed identity after deployment, if desired.\n\n[Learn more about Azure Sentinel](https://aka.ms/azuresentinel)",
```

Comment:
@v-mchatla, please change text from Azure Sentinel to Microsoft Sentinel, thanks.